README.md

```diff
@@ -1 +1,37 @@
-# code-runner
\ No newline at end of file
+Code Runner: An Environment to run your code
+===================================================
+Code Runner is a web application to run code snippet or code files on a docker environment with code specific configurations.
```

Comment:
**A web application to run code pertaining to your assignment using user provided configurations.**

Reply:
Done